src/orchestrator/orchestrator.rs

```diff
@@ -562,15 +567,18 @@ impl<'a> Drop for JobTask<'a> {
             let errmsg = if self.jobdef.dependencies.is_empty() {
                 "error occurred"
             } else {
-                "error on other task"
+                "inherited"
             };
 
-            self.bar.finish_with_message(format!(
-                "[{} {} {}] Stopped, {msg}",
+            self.bar.set_message(format!(
+                "{:-<MAX_ENDPOINT_NAME_LENGTH$} {:-<CONTAINER_ID_LENGTH$} {:-<JOB_UUID_LENGTH$} {} {} {} Stopped, {msg}",
+                "", // endpoint_name
+                "", // container_id
                 self.jobdef.job.uuid(),
+                "██".red(),
                 self.jobdef.job.package().name(),
                 self.jobdef.job.package().version(),
-                msg = errmsg
+                msg = errmsg.yellow()
```

Comment:
Question: Should the error message really be yellow instead of red? Is this because of the inherited/propagated errors?